IndexTestFunctions.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
This file should be called functions.php

This file should contain most of the php that is currently in index.php